includes/functions.php

```diff
@@ -2524,3 +2524,94 @@ function oxidized_node_update($hostname, $msg, $username = 'not_provided')
     }
     return false;
 }//end oxidized_node_update()
+
+
+/**
+ * @params int code
+ * @params int subcode
+ * @return string
+ * Take a BGP error code and subcode to return a string representation of it
+ */
+function describe_bgp_error_code($code, $subcode)
+{
+    // https://www.iana.org/assignments/bgp-parameters/bgp-parameters.xhtml#bgp-parameters-3
+    $codes = [
```

Comment:
The best way to do this is to create a new language file (resources/lang/en/bgp.php) and add all these messages there, that way you can look them up like `trans('bgp.error_codes.$code")`  It will also allow translation :)

Reply:
Done